CHANGELOG.md

```diff
@@ -217,12 +190,12 @@ Those changes in added, changed or breaking changes, should include usage exampl
 
 * `TaskMetadata.task_created_block` field changed to `u64` [#362](https://github.com/Layr-Labs/eigensdk-rs/pull/362)
 
-* Separated the interface and service in the `bls_agg` module in [#363](https://github.com/Layr-Labs/eigensdk-rs/pull/363).
```

Comment:
This entry got reverted

Reply:
https://github.com/Layr-Labs/eigensdk-rs/pull/381/commits/47b5051ce55d65474f6072a1db35eaa82df23dbd